description.md

```diff
@@ -18,8 +17,34 @@ Each topic in this file contains a small description and links to the files cove
 
 ### Assignments
 
-### Drills
+Assignments are practical activities that learners undertake individually or in teams over the span of multiple weeks.
```

Comment:
You should specify that they may be both graded and ungraded, but most of the time they are graded to incentivise students.